tests/queries/0_stateless/01651_lc_insert_tiny_log.sql

```diff
@@ -0,0 +1,15 @@
+drop table if exists perf_lc_num;
+
+CREATE TABLE perf_lc_num(　        num UInt8,　        arr Array(LowCardinality(Int64)) default [num]　        ) ENGINE = TinyLog;
```

Comment:
Maybe copy-paste this test for Log, StripeLog - just in case?

Reply:
Done